xml/oai2.xsl

```diff
@@ -598,7 +598,7 @@ p.intro {
   <xsl:choose>
     <xsl:when test='starts-with(.,"http" )'>
       <xsl:choose>
-        <xsl:when test='string-length(.) &gt; 50'>
+        <xsl:when test='string-length(.) &gt; 80'>
```

Comment:
Is this change intentional? It's 3rd-party code, which we try not to distribute modified copies of if possible.